app/views/publishers/_choose_channel_type.html.slim

```diff
@@ -0,0 +1,27 @@
+= link_to(new_site_channel_path, class: 'card block-link--card') do
```

Comment:
this link doesn't actually work for me. it dismisses the modal and prevents default

Reply:
Addressed. In https://github.com/brave-intl/publishers/commit/1c60fc7f8a877441ab48397f3c27cc5170906522#diff-fdc082bd84b5d4eea4238f264d49e385R91 the `preventDefault` call was moved to incorrectly be fired any time the user clicked anywhere that was not the background of the page. Instead, `preventDefault` and other closing logic should only be called if `js-confirm` or `js-deny` is clicked.